kymatio/scattering1d/__init__.py

```diff
@@ -1,4 +1,3 @@
 __all__ = ['Scattering1D']
```

Comment:
should be at the end, as in https://github.com/kymatio/kymatio/blob/v0.2/kymatio/scattering2d/__init__.py